tests/end_to_end/test_helper.cpp

```diff
@@ -244,7 +244,7 @@ GlueCreateFileW(
     return (HANDLE)CREATE_FILE_HANDLE;
 }
 
-bool
+BOOL
```

Comment:
curious why there's no SAL `_Success_(return ...` annotation even though the signature _was_ modified (return value changed from `bool` to `BOOL`)?  Same for `GlueCancelIoEx` below.